docs/source/skcosmo.preprocessing.rst

```diff
@@ -0,0 +1,11 @@
+Flexible Preprocessing
```

Comment:
Can we call this file `preprocessing`? No need to name it `skcosmo`, otherwise everything would start `skcosmo.`